lib/app.ts

```diff
@@ -24,6 +24,8 @@ import {
 } from './preparser';
 import { CliSettings } from './utils/bootstrap';
 import { onceAsync } from './utils/lazy';
+import events = require('events');
+import * as BalenaSdk from 'balena-sdk';
```

Comment:
@myarmolinsky I don't think we want/can move these imports to top level. This means that this code will be loaded everytime (even when it is not used) as long as any function is imported, which can impact the load time of the CLI. We want to have this as an import or await import but inside the function where it is used.